### PR TITLE
Design Improvements

### LINKED ISSUE #144: Mask Roles on Profile Page
I'm not sure what the other roles look like, but as a User, my role appears as ROLE_USER.  Maybe mask to just say "User"?

image

### LINKED ISSUE #258: Clicking the name of a filter won't activate it
Clicking on the text of for instance 'Stone Giants' und "Common Monsters" doesn't toggle the filter. Only clicking next to the filter does. Probably need to move the click handler?

### LINKED ISSUE #264: Regression(?): The .always-opens aren't always open
On page load both locally and in dev.adventurelookup, the filters that have classes `.always-open` aren't open. Are they not supposed to be any more?

### LINKED ISSUE #266: Incomplete Level Range not Displaying
It appears that adventures which lack entries in both maximum and minimum level, but have an entry in one will not display that level range on the search results page.

This adventure has a minimum level of 7:
advpage

But on the search results screen there is nothing to indicate that:
nonshow

Since we've got this information it seems like a waste not to display it. Perhaps incomplete level designations like this should be displayed as "7-?" or just as "7"?

### LINKED ISSUE #269: Display issue with filter drop downs
As you can see in this image the drop downs exceed their region.

screen shot 2018-04-26 at 6 32 00 pm

### PR DESCRIPTION
**Pull request by cmfcmf**
This PR fixes several issues with the new design. Most importantly, this makes the new design usable on mobile devices and fixes multiple reported and un-reported issues (fixes #269, fixes #264, fixes #258, fixes #266, fixes #144). I also added a cookie consent warning (let me know if you have feedback regarding that). I'm not too happy with how the adventure detail page looks in between mobile- and desktop-sized viewports. If you have any feedback regarding that, I'll try to improve it further.

This PR should bring us a lot closer to actually rolling out the new design.

### CODE REVIEW

File: app/Resources/views/adventure/index.html.twig [hunk 1]
```diff
@@ -4,7 +4,8 @@
 {% block body %}
     <div class="container">
         <div class="row">
-            <div class="col-1">
+            {# ADL sidebar #}
+            <div class="col-3 mt-5           col-sm-2                     col-lg-1">
```
Comment (burnhamrobertp) [inline app/Resources/views/adventure/index.html.twig]: What's up w/ the whitespace here between the classnames?

File: app/Resources/views/adventure/index.html.twig [hunk 2]
```diff
@@ -17,101 +18,109 @@
                     </div>
                 </div>
             </div>
-            <div class="col-9">
+            {# Content #}
+            <div class="col-12 order-last                                 col-lg-9 order-lg-2">
```
Comment (burnhamrobertp) [inline app/Resources/views/adventure/index.html.twig]: whitespace peculiarity in classes

File: app/Resources/views/adventure/index.html.twig [hunk 3]
```diff
@@ -17,101 +18,109 @@
...
-                                        </button>
-                                    {% else %}
-                                        <button class="dropdown-item disabled" type="button">Please create a list first, using the link below</button>
-                                    {% endfor %}
-                                    <div class="dropdown-divider"></div>
-                                    <a class="dropdown-item" href="{{ path('adventure_lists_index') }}">
-                                        <i class="fa fa-fw fa-cog"></i>
-                                        Manage lists
+            {# Buttons #}
+            <div class="col-9 mt-5           col-sm-10                    col-lg-2 order-lg-3 mt-lg-5">
```
Comment (burnhamrobertp) [inline app/Resources/views/adventure/index.html.twig]: whitespace peculiarity in classes

File: app/Resources/views/adventure/index.html.twig [hunk 4]
```diff
@@ -17,101 +18,109 @@
...
+            <div class="col offset-lg-1">
+                <h2 class="subheading mt-5">From the community</h2>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-xs-12 order-last col-md-9 order-md-1 offset-lg-1">
                 {% include 'adventure/_reviews.html.twig' %}
             </div>
-            <div class="col-2">
+            <div class="col-xs-12            col-md-3 order-md-2 col-lg-2 mb-5 mb-md-0">
```
Comment (burnhamrobertp) [inline app/Resources/views/adventure/index.html.twig]: whitespace peculiarity in classes

**Comment by cmfcmf** [comment]
I've now removed the extraneous whitespace; it was left over from tinkering with different combinations of classes and breakpoints.

**Pull request merged**